src/App.jsx

```diff
@@ -28,6 +29,7 @@ class App extends Component {
             <Route exact path="/verify/:token" component={verify} />
             <Route path="/getprofile" component={Profile} />
             <Route path="/edit-profile" component={EditProfile} />
+            <Route path="/profiles" component={UserProfiles} />
```

Comment:
This can be made exactly to avoid making this url render profile 
`/profiles/hdhdhjdjdjdjdjdjdhdjjdjdjdj`
use `<Route exact path="/profiles" component={UserProfiles} />`